src/current/v23.2/column-level-encryption.md

```diff
@@ -0,0 +1,327 @@
+
+Column level encryption can be useful for compliance scenarios such as adhering to PCI or GDPR.
+
+Once a column is encrypted, SQL users are required to pass an encryption key when reading from or writing to that column. This prevents users without the encryption key from taking actions like the following:
+
+- Reading sensitive information (like _personally identifiable information_ ([PII](https://en.wikipedia.org/wiki/Personal_data))) when accessing the tables.
+- Adding or updating records without encrypting the specified columns.
+
+{{site.data.alerts.callout_info}}
+The "column-level encryption" feature described on this page specifically refers to either symmetric-key encryption or public-key encryption of data, **not** hashing functions operating on data.
+{{site.data.alerts.end}}
+
+{% include enterprise-feature.md %}
+
+## Available Functions
+
+### `encrypt` and `encrypt_iv`
+
+The `encrypt` and `encrypt_iv` functions encrypts a column's data with a given key and cipher method. For more information, see [Cryptographic functions]({% link {{ page.version.version }}/functions-and-operators.md %}#cryptographic-functions).
+
+For usage examples, see:
+
+- [Encrypt using the `encrypt` function](#encrypt-using-the-encrypt-function).
+- [Encrypt using the `encrypt_iv` function](#encrypt-using-the-encrypt_iv-function).
+
+These functions require an [enterprise license]({% link {{ page.version.version }}/enterprise-licensing.md %}). If you do not have an enterprise license, you will see an error message like the following if you try to use them:
+
+~~~
+ERROR:  encrypt(): use of pgcrypto cipher functions requires an enterprise license. see https://cockroachlabs.com/pricing for details on how to enable enterprise features
+~~~
+
+{{site.data.alerts.callout_info}}
+For more information about whether to use the `encrypt` or `encrypt_iv` variants of this function, see [Whether to use `encrypt` or `encrypt_iv`?](#whether-to-use-encrypt-and-decrypt-or-encrypt_iv-and-decrypt_iv).
+{{site.data.alerts.end}}
+
+### `decrypt` and `decrypt_iv`
+
+The `decrypt` and `decrypt_iv` functions decrypt an encrypted column's data with a given key and cipher method. For more information, see [Cryptographic functions]({% link {{ page.version.version }}/functions-and-operators.md %}#cryptographic-functions).
+
+For usage examples, see:
+
+- [Decrypt using the `decrypt` function](#decrypt-using-the-decrypt-function).
+- [Decrypt using the `decrypt_iv` function](#decrypt-using-the-decrypt_iv-function).
+
+These functions require an [enterprise license]({% link {{ page.version.version }}/enterprise-licensing.md %}). If you do not have an enterprise license, you will see an error message like the following if you try to use them:
+
+~~~
+ERROR:  decrypt(): use of pgcrypto cipher functions requires an enterprise license. see https://cockroachlabs.com/pricing for details on how to enable enterprise features
```

Comment:
The error message has changed, see above.

Reply:
updated to use the error message from your other comment